pkg/pb/metainfo.proto

```diff
@@ -196,3 +203,129 @@ message ProjectInfoRequest {
 message ProjectInfoResponse {
     bytes project_salt = 1;
 }
+
+message Object {
```

Comment:
Is this a replacement for the `Pointer` protobuf? If yes, where is the info for remote pieces stored?

Reply:
`Segment` is part of `Object` and it contains `nodes`

Reply:
Where can I see the `Segment` protobuf? I also don't see any reference to segments here in the `Object` protobuf. Is this `stream_id`?

Reply:
its in design doc https://github.com/storj/storj/blob/6aefa1ae809d9df083f36f58a812bce91e68380c/docs/design/metainfo-rpc.md#L222

Reply:
Are we going to introduce it with a separate PR?

Reply:
yes, I'm working on that but its hard to push it without changes from this PR

Reply:
:+1: